WISE-Community/WISE-Client
# feat(Component Info): Allow multiple examples per component type

## Changes
- Allow multiple examples per component type in the component info popup
- If a component type has multiple examples, it will display tabs
- If a component type only has one example, it will not display tabs
- Multiple Choice now has two examples but all others still only have one
- Updated all component type info classes to contain an array of preview examples even if they only have one example

## Test
- Make sure Multiple Choice shows two examples
- Make sure all other component types only show one example

Closes #1519
